### DISCUSSION TITLE
MDOC filename filter not working

### DISCUSSION THREAD

**Discussion by asarnow** [nextpyp/.github#104] [Q&A] [answered]
There is an issue with the MDOC filename filter: it seems all of the MDOC files are processed even those excluded by the pattern. For example, there are names like `rd6_ts_020.mrc.mdoc` which are real tilt-series MDOC files, as well as `rd6_tgt_002.mrc.mdoc` and `rd6_tgt_001_view.mrc.mdoc` which are used internally by PACE. Using the filter `*ts*.mrc.mdoc` appears to show only the desired tilt-series MDOCs, but the preprocessing logs show that all of the MDOCs are used. 

It took me a while to figure out what was going on, those other MDOC files cause nextPYP to output a lot of error messages for apparent failed tilt-series, with the incorrect DateTime format exception, which led me in the wrong direction until I read all the logging output carefully.

**Comment by tofushan** [comment, marked as answer]
Thanks for reporting! We've fixed the issue with the file pattern for mdoc files. You should be able to filter out bad tilt-series based on the mdoc file pattern in the next release.

**Reply by abartesaghi** [reply]
This issue was fixed in version [0.6.5](https://nextpyp.app/files/pyp/0.6.5/docs/index.html).
